src/test/java/com/example/demo/ConnectionUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.example.demo;
+
+import org.jooq.DSLContext;
+import org.jooq.impl.DSL;
+
+public class ConnectionUtils {
```

Comment:
class only with static methods. Please add set access level private for constructor